rasa/core/policies/embedding_policy.py

```diff
@@ -548,6 +552,12 @@ def tf_feed_dict_for_prediction(
 
         return {self.a_in: session_data["dialogue_features"][0]}
 
+    def _normalize_scores(self, confidence):
```

Comment:
good idea to create a function, could you please move it to `train_utils.py` then it can be reused in nlu as well